website/content/docs/commands/credentials/update.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: doc
 page_title: credentials update - Command
-description: |-
-  The "credentials update" command lets Boundary admin update a credential resource.
+description: >-
+  The "credentials update" command updates an existing credential resource.
```

Comment:
I'm noticing the lack of "for a given ID" throughout certain command that require it, like this one and others, like delete.

Do we need to specify "for a given ID" at all? I think it could be removed, or we should standardize and ensure all the commands that require it have it

Reply:
Yeah, I think it was in the template I used for the `read` commands, but not the others. I agree it is probably not essential for understanding what the command does. I'm fine with either adding it or removing it throughout if you have a strong feeling one way or the other, but I think I would lean towards removing it just so it doesn't become an issue with the 160 character count on the longer descriptions.

Reply:
let's remove it - i don't think it provides much value, and will take less time to implement

Reply:
Sounds good! I will update these and let you know when it's ready for another look.